_docs/style/text-conventions/alert-messages.md

```diff
@@ -0,0 +1,291 @@
+---
+title: Alert and Message Guidelines
+layout: docs
+parent: Text Conventions
+category: Style
+usage: |
+  Alerts and messages inform users about progress that they make or problems that they encounter in the UI. They must be clearly written, be free from grammar and punctuation problems, and follow the style and terminology guidelines in this section. Users gain understanding from the content of the messages (text, icons, color) as well as the context (where on the screen and when in the user flow the messages appear). So users can recover quickly, alerts and messages should provide essential information to help them understand and address issues.
+resource: false
+status: complete
+last-modified: 2018-06-28
+---
+
+{% include toc.html %}
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-12-md hxSpan-12-lg" %}
+
+## General guidelines
+
+When you write alerts and messages, consider the following guidelines:
+
+- Be courteous and do not blame the user.
+- Use present tense to describe conditions that currently exist, or use past tense to describe a specific event that occurred in the past.
+- Where possible, guide users with the imperative voice (for example, "Enter a valid email."") or the active voice (such as "The Control Panel is not responding.").
+- Alerts and messages must be short, accurate, complete, and helpful.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-12-md hxSpan-12-lg" %}
+
+## Message guidelines and examples
+
+Use the following guidelines when you write messages in a control panel.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+Use complete sentences, when possible. Include articles (a, an, the) to make the sentence complete. If possible, use active voice.
+
+**Note**: Message text that serves as a heading or label (such as **Elapsed:hh:mm:ss**, which indicates elapsed time) is acceptable as a sentence fragment.
```

Comment:
I'm not sure i understand this example. How is elapsed time a heading or a label?